README.md

```diff
@@ -1 +1,13 @@
-the project is about .....
+awk -F',' '$1 ~ /^16:17/ { print }' logfile.csv > time.csv
+cat time.csv
+we get all 16:17 om the page
+---------
+awk -F',' '$2 != "INFO" time.csv > noinfo.csv
+cat noinfo.csv
+we put all the lines exept the lines that have info to the noinfo file
+------------
+cat noinfo.csv | grep WARN | wc -l
```

Comment:
the result you provided is not equal to the expected result 
the result should be the count of warnings per each class